frontend/src/features/admin-form/create/builder-and-design/mutations/useDeleteFormField.ts

```diff
@@ -85,6 +106,34 @@ export const useDeleteFormField = () => {
     [toast],
   )
 
+  const deletePaymentFieldMutation = useMutation(
+    () => updateFormPayments(formId, { enabled: false } as PaymentsUpdateDto),
```

Comment:
This is a little yucky due to the types.

I'm thinking if it is ok to set our default as `{ enabled: false, paymentType: PaymentType.Fixed, amount_cents: 0 }` which also reflects the DB defaultd -- couldn't really think of anything better 😢

Reply:
On a related note of removing forced type casting, could I also get your help to remove `as FormPaymentsField` in [frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/PaymentsInputPanel/PaymentsInputPanel.tsx:L126](https://github.com/opengovsg/FormSG/blob/feat/migrate-payment-enablement/frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/PaymentsInputPanel/PaymentsInputPanel.tsx#L126C12-L126C29)?

Reply:
Its still a little yucky as I had to hard-type `payment_type` else typescript will complain 😩 [62baf5c](https://github.com/opengovsg/FormSG/pull/6511/commits/62baf5c825d6b37bab5a3dc513e2205bb6cdb379)

Reply:
[85f399d](https://github.com/opengovsg/FormSG/pull/6511/commits/85f399d63aefb6752741e535943f1157b57215c2) Paired thanks Ken!